src/vs/workbench/contrib/preferences/browser/settingsTree.ts

```diff
@@ -381,9 +381,9 @@ export async function resolveExtensionsSettings(extensionService: IExtensionServ
 		let sectionOrder = 0;
 		const extensionConfiguration = extension?.contributes?.configuration;
```

Comment:
I guess I was thinking that we shouldn't be looking up the configuration off the extension contribution, since aren't we sort of re-looking up the configuration that we already have on ISettingsGroup? Should that have the order?

Reply:
I took a look and noticed `configurationRegistry.ts`. It seems `IConfigurationNode` has all the fields I need, but I'll have to see how that interface is being translated to `ISettingsGroup`, and how it's being populated in the first place. I set up a breakpoint and I think `order` wasn't preserved in the `IConfigurationNode` objects.

Reply:
Got this fixed.